### PR TITLE
V4 Copy Review Odds & Ends

### PR DESCRIPTION
**Pull request by adghayes**

### CODE REVIEW

File: apps/scan/frontend/src/screens/polls_not_open_screen.tsx [hunk 1]
```diff
@@ -36,6 +36,8 @@ export function PollsNotOpenScreen({
       >
         {pollsState === 'polls_closed_final' ? (
           <P>Voting is complete.</P>
+        ) : pollsState === 'polls_paused' ? (
```
Comment (adghayes) [inline apps/scan/frontend/src/screens/polls_not_open_screen.tsx:39]: Screen Shot 2024-10-22 at 8 55 35 AM
Screen Shot 2024-10-22 at 8 57 24 AM

File: apps/scan/frontend/src/screens/poll_worker_shared.tsx [hunk 2]
```diff
@@ -22,13 +22,13 @@ export function getPostPollsTransitionHeaderText(
...
+      return 'Polls Closed';
     case 'open_polls':
-      return 'Polls are open.';
+      return 'Polls Opened';
     case 'resume_voting':
-      return 'Voting resumed.';
+      return 'Voting Resumed';
     case 'pause_voting':
-      return 'Voting paused.';
+      return 'Voting Paused';
```
Comment (adghayes) [inline apps/scan/frontend/src/screens/poll_worker_shared.tsx:31]: Screen Shot 2024-10-22 at 8 52 53 AM
Screen Shot 2024-10-22 at 8 52 18 AM
Screen Shot 2024-10-22 at 8 51 43 AM
Screen Shot 2024-10-22 at 8 51 27 AM
Screen Shot 2024-10-22 at 8 50 45 AM
Screen Shot 2024-10-22 at 8 50 26 AM
Screen Shot 2024-10-22 at 8 50 04 AM
Screen Shot 2024-10-22 at 12 18 05 AM

File: libs/ui/src/invalid_card_screen.tsx [hunk 3]
```diff
@@ -74,6 +73,9 @@ export function InvalidCardScreen({
       errorDescription =
         `The inserted card’s jurisdiction (${cardJurisdiction}) does not match ` +
         `this machine’s jurisdiction (${machineJurisdiction}).`;
+      // If the card is the wrong jurisdiction, there's probably not an action for a
+      // typical customer to take. The warning is mostly for our own testing and production.
+      recommendedAction = '';
```
Comment (adghayes) [inline libs/ui/src/invalid_card_screen.tsx:78]: Screen Shot 2024-10-22 at 9 24 40 AM
Screen Shot 2024-10-22 at 9 50 32 AM

Addresses #5200.

File: apps/scan/frontend/src/screens/poll_worker_fujitsu_post_print_screen.tsx
[the remark this answers is not in the document]
Comment (jonahkagan) [reply apps/scan/frontend/src/screens/poll_worker_fujitsu_post_print_screen.tsx:1]: We should consider right-aligning text here. Centered text that wraps onto multiple lines is hard to read

**Review by jonahkagan — APPROVED** [review]

File: apps/scan/frontend/src/screens/poll_worker_fujitsu_post_print_screen.tsx
[the remark this answers is not in the document]
Comment (adghayes) [reply apps/scan/frontend/src/screens/poll_worker_fujitsu_post_print_screen.tsx:1]: That's a good principle I've never heard before! I'm going to ship this but I do agree. @mattroe and I didn't have a big focus on #5182 during our review.

**Pull request merged**